lametro/templatetags/lametro_extras.py

```diff
@@ -244,7 +244,11 @@ def get_list(querydict, key):
 @register.filter
 def get_bill_type_link(bill_type):
     for legislation_type in LEGISLATION_TYPE_DESCRIPTIONS:
-        if legislation_type['search_term'].upper() == bill_type.upper():
+        if legislation_type['search_term'].title() == bill_type:
```

Comment:
Can you say more about this change?

Reply:
Looking back over this, I think it's probably an unnecessary change. I just used `title()` because the `bill_type` is usually in title case.

It might actually be better to use `upper()` to avoid any weird edge cases down the line.

Reply:
Yeah, I like to superstitiously apply the same transformation on both sides so there are no capitalization mishaps during comparison.